src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int maxNumber = 100;
```

Comment:
constants should be uppercased
```suggestion
    private static final int MAX_NUMBER = 100;
```